custom_components/ramses_cc/services.yaml

```diff
@@ -443,23 +444,23 @@ set_dhw_mode:
         boolean:
 
     duration:
-      name: Override duration
+      name: Duration
       description: >-
         The duration of the temporary_override. Mutually exclusive with until.
-      example: '{"minutes": 135}'
+      default:
+        hours: 1
+        minutes: 30
+      example: '{"hours": 1, "minutes": 30}'
       selector:
-        number:
-          min: 15
-          max: 1440
-          step: 15
-          unit_of_measurement: mins
-          mode: slider
+        duration:
 
     until:
-      name: Override end time
+      name: Until
```

Comment:
Renamed so the references in the description regarding mutual exclusivity make sense.